client/src/plugins/user-journey-statistics/event-handlers/__tests__/OverlayEventHandlerSpec.js

```diff
@@ -0,0 +1,448 @@
+
+          // when
+          await handleOverlayAction({
+            tab,
+            context: 'deploymentTool'
+          });
+
+          // then
+          expect(track).to.have.been.calledWith('overlay:deploy:opened', {
+            diagramType: 'dmn'
+          });
+        });
+
+
+        it('cloud dmn', async () => {
+
+          // given
+          const tab = createTab({
+            type: 'cloud-dmn'
+          });
+
+          const handleOverlayAction = subscribe.getCall(0).args[1];
+
+          // when
+          await handleOverlayAction({
+            tab,
+            context: 'deploymentTool'
+          });
+
+          // then
+          expect(track).to.have.been.calledWith('overlay:deploy:opened', {
+            diagramType: 'dmn'
+          });
+
+        });
+
+      });
+
+
+      describe('start instance tool', () => {
+
+        it('bpmn', async () => {
+
+          // given
+          const tab = createTab({
+            type: 'bpmn'
+          });
+
+          const handleOverlayAction = subscribe.getCall(0).args[1];
+
+          // when
+          await handleOverlayAction({
+            tab,
+            context: 'startInstanceTool'
+          });
+
+          // then
+          expect(track).to.have.been.calledWith('overlay:startInstance:opened', {
+            diagramType: 'bpmn'
+          });
+        });
+
+
+        it('cloud bpmn', async () => {
+
+          // given
+          const tab = createTab({
+            type: 'cloud-bpmn'
+          });
+
+          const handleOverlayAction = subscribe.getCall(0).args[1];
+
+          // when
+          await handleOverlayAction({
+            tab,
+            context: 'startInstanceTool'
+          });
+
+          // then
+          expect(track).to.have.been.calledWith('overlay:startInstance:opened', {
+            diagramType: 'bpmn'
+          });
+        });
+
+
+        it('dmn', async () => {
+
+          // given
+          const tab = createTab({
+            type: 'dmn'
+          });
+
+          const handleOverlayAction = subscribe.getCall(0).args[1];
+
+          // when
+          await handleOverlayAction({
+            tab,
+            context: 'startInstanceTool'
+          });
+
+          // then
+          expect(track).to.have.been.calledWith('overlay:startInstance:opened', {
+            diagramType: 'dmn'
+          });
+        });
+
+
+        it('cloud dmn', async () => {
+
+          // given
+          const tab = createTab({
+            type: 'cloud-dmn'
+          });
+
+          const handleOverlayAction = subscribe.getCall(0).args[1];
+
+          // when
+          await handleOverlayAction({
+            tab,
+            context: 'startInstanceTool'
+          });
+
+          // then
+          expect(track).to.have.been.calledWith('overlay:startInstance:opened', {
+            diagramType: 'dmn'
+          });
+        });
+
+      });
+
+
+    });
+
+
+    describe('should send engine profile', () => {
```

Comment:
Could we test here that this profile is actually sent (just as in the other tests)? E.g.,
```
      expect(track).to.have.been.calledWith('overlay:versionInfo:opened', {
        executionPlatform: 'Camunda Platform'
      });
```

Or is there any reason not to do that?

Reply:
I saw it diffing from the other test cases. But you are right, for `executionPlatform` we always test using this pattern (also in existing other test cases).